lms/envs/common.py

```diff
@@ -1291,6 +1295,18 @@
         ],
         'output_filename': 'css/lms-style-course-vendor.css',
     },
+    'style-edx-footer': {
+        'source_filenames': [
+            'sass/footer-v3.css',
```

Comment:
Any way we can name this file so its edX-specific (assuming it only contains edx.org-related styling)?